src/components/app-configure/ConfigureActionButtons.vue

```diff
@@ -0,0 +1,62 @@
+<script lang="ts" setup>
+import VButton from '@/components/lib/VButton/VButton.vue'
+
+defineProps({
+  saveLabel: {
+    type: String,
+    default: 'SAVE',
+  },
+  cancelLabel: {
+    type: String,
+    default: 'CANCEL',
+  },
+  saveDisabled: {
+    type: Boolean,
+    default: false,
+  },
+  cancelDisabled: {
+    type: Boolean,
+    default: false,
+  },
+})
+
+const emit = defineEmits(['cancel', 'save'])
+</script>
+
+<template>
+  <div class="footer flex flex-wrap">
+    <VButton
+      v-wave
+      type="button"
+      :label="cancelLabel"
+      variant="secondary"
+      :disabled="cancelDisabled"
+      @click.stop="emit('cancel')"
+    />
+    <VButton
+      v-wave
+      type="submit"
+      :label="saveLabel"
+      variant="primary"
+      :disabled="saveDisabled"
+      @click.stop="emit('save')"
+    />
+  </div>
+</template>
+
+<style scoped>
+.footer {
+  display: flex;
+  gap: 2rem;
+  justify-content: flex-end;
```

Comment:
- We already have `flex` on the footer div.
- Can the gap and justify-content lines be replaced with utility classes?

Reply:
For justify currently we only have utility classes for space-between, space-around, space-evenly and center. I didn't add new classes since we are moving to tailwind

For gap we don't have utility classes, we use VStack with gap prop

@ajithranka